xrpl4j-address-codec/src/test/java/org/xrpl/xrpl4j/codec/addresses/XAddressTest.java

```diff
@@ -220,35 +219,23 @@ public void convertBetweenClassicAndXAddressTestnet(
   }
 
   @Test
-  public void xAddressWithBadChecksum() {
-    XAddress xAddress = XAddress.of("XVLhHMPHU98es4dbozjVtdWzVrDjtV5fdx1mHp98tDMoQXa");
-
-    assertThrows(
-      EncodingFormatException.class,
-      () -> addressCodec.xAddressToClassicAddress(xAddress),
-      "Checksum does not validate"
-    );
-  }
-
-
-  @Test
-  public void xAddressWithBadPrefix() {
-    XAddress xAddress = XAddress.of("dGzKGt8CVpWoa8aWL1k18tAdy9Won3PxynvbbpkAqp3V47g");
+  public void xaddressWithBadPrefix() {
+    XAddress xaddress = XAddress.of("dGzKGt8CVpWoa8aWL1k18tAdy9Won3PxynvbbpkAqp3V47g");
 
     assertThrows(
       DecodeException.class,
-      () -> addressCodec.xAddressToClassicAddress(xAddress),
+      () -> addressCodec.xAddressToClassicAddress(xaddress),
       "Invalid X-Address: Bad Prefix"
     );
   }
 
   @Test
-  public void xAddressWith64BitTag() {
-    XAddress xAddress = XAddress.of("XVLhHMPHU98es4dbozjVtdWzVrDjtV18pX8zeUygYrCgrPh");
+  public void xaddressWith64BitTag() {
```

Comment:
Why the rename here?

Reply:
It was not following the regex in `AddressCodecTest` so I changed here as well

Reply:
Can you revert this, and add this to `AddressCodecTest`?

```
@SuppressWarnings( {"ParameterName", "MethodName", "LocalVariableName"})
public class AddressCodecTest {
```

Reply:
Okay!